kyuubi-ctl/src/main/scala/org/apache/kyuubi/ctl/cmd/create/CreateBatchCommand.scala

```diff
@@ -43,13 +43,15 @@ class CreateBatchCommand(cliConfig: CliConfig) extends Command[Batch](cliConfig)
       val request = map.get("request").asInstanceOf[JMap[String, Object]]
       val config = request.get("configs").asInstanceOf[JMap[Object, Object]].asScala
         .map { case (k, v) => (k.toString, v.toString) }.asJava
+      val args = request.get("args").asInstanceOf[ArrayList[Object]].asScala
+        .map(x => x.toString).asJava
       val batchRequest = new BatchRequest(
         map.get("batchType").asInstanceOf[String],
```

Comment:
should the  batchType from request？

Reply:
```
request.get("batchType").asInstanceOf[String]
```

Reply:
BTW, the batchType in https://github.com/apache/incubator-kyuubi/blob/master/kyuubi-ctl/src/test/resources/cli/batch.yaml is missing

Reply:
![image](https://user-images.githubusercontent.com/6757692/176089575-bf919bd4-df9c-4420-b1e6-5c49e0a40095.png)

cc @ulysses-you

Reply:
how about read from `map` first, if not defined, then fall back to `request`.

Reply:
how can we fall back to request, do you mean  ?
```yml
request:
  name:
  batchType:
```

Reply:
yes.

Because it is a filed of dto `BatchRequest`.

Reply:
seems we do not need read from map first, just get it from request